hooks/pre/30-upgrade.rb

```diff
@@ -72,7 +72,7 @@ def fix_katello_settings_file
 
   return true unless settings.key?(:common)
 
-  settings = {:katello => settings[:common]}
+  settings = { :katello => settings[:common] }
```

Comment:
That is odd but it seems that's [what rubocop does by default](https://git.io/v6x9L). I'm going to copy over the setting from katello: https://git.io/v6x9s